src/lab1/structures/vertex/Vertex3D.ts

```diff
@@ -12,6 +16,30 @@ export default class Vertex3D {
     this.z = z;
   }
 
+  public translate(x: number, y: number, z: number): Vertex3D {
+    const tranformed = transformVector(
+      this.toVector(),
+      transformations.translate3d(x, y, z)
+    );
+    return tranformed.toVertex3D();
+  }
+
+  public rotate(angleX: number, angleY: number, angleZ: number): Vertex3D {
+    const tranformed = transformVector(
+      this.toVector(),
+      transformations.rotate3d(angleX, angleY, angleZ)
+    );
+    return tranformed.toVertex3D();
+  }
+
+  public scale(x: number, y: number, z: number): Vertex3D {
+    const tranformed = transformVector(
+      this.toVector(),
+      transformations.scale3d(x, y, z)
+    );
+    return tranformed.toVertex3D();
+  }
```

Comment:
We can't scale or rotate a vertex due to its physical nature (vertex doesn't have any sizes, only coordinates). If you use this methods, you are probably doing something wrong
```suggestion
```